<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scoresMenuClass</class>
 <widget class="QWidget" name="scoresMenuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Titre">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Scores</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_Parcours">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>571</width>
     <height>211</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Pseudo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Score</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nom Parcours</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
